SYCL/lit.cfg.py

```diff
@@ -92,6 +92,10 @@
            llvm_config.with_environment(var,"")
 
 config.substitutions.append( ('%sycl_libs_dir',  config.sycl_libs_dir ) )
+if platform.system() == "Windows":
+    config.substitutions.append( ('%sycl_static_libs_dir',  config.sycl_libs_dir + '/../lib' ) )
```

Comment:
Should this path be using `\` rather than `/` since it is for windows, or are we guaranteed that the correct conversion will happen?

Reply:
Other settings using `/` which are applied for the Windows platform look to be resolving as expected. For example, the `%sycl_options` resolution of the `sycl.lib` library for linking.

Reply:
In that case it is likely fine.